contentwarehouse/snippets/update_document_schema_sample.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+# [START contentwarehouse_update_document_schema]
+
+from google.cloud import contentwarehouse
+
+# TODO(developer): Uncomment these variables before running the sample.
+# project_number = "YOUR_PROJECT_NUMBER"
+# location = "us" # Format is 'us' or 'eu'
+# document_schema_name = "YOUR_SCHEMA_NAME" # Format is "projects/YOUR_PROJECT_NUMBER/locations/us/documentSchemas/SCHEMA_NAME"
```

Comment:
It probably makes more sense for this to be `document_schema_id` since the rest of the elements can be inferred from project_number and location.

Then you can construct the full document_schema_name using document_schema_client.document_schema_path or something similar